initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -898,6 +898,17 @@
              name="s1"
              netAddress="."/>
   </EclSchedulerProcess>
+  <TopoServerProcess build="_"
+                     buildSet="toposerver"
+                     daliServers=""
```

Comment:
Should probably be daliServers="mydali" to match all other dali references in other components

Reply:
I don't think the toposerver connects to dali

Reply:
Removed references to daliserver from the TopoServerProcess